lib/strapi/search.ts

```diff
@@ -0,0 +1,146 @@
+'use server'
+import qs from 'qs'
+import { fetchCollection, SingleResponse } from '.'
+import { PageType, Section } from '@models/page'
+import { publicDrive, privateDrive } from '@lib/google/drive'
+import { getServerSession } from 'next-auth'
+import { authOptions } from '@lib/nextauth'
+import { drive_v3 } from 'googleapis'
+
+export type SearchData = {
+  sectionData: Section[]
+  pageData: SingleResponse<PageType>[]
+  privateSectionData: Section[]
+  privatePageData: SingleResponse<PageType>[]
+}
+
+export async function searchPublic(
```

Comment:
Since it now also searches private content
```suggestion
export async function searchSiteContent(
```